src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -14,4 +18,11 @@
     @ManyToOne
     @JoinColumn(name = "group_id")
     public Group group;
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "join_orders_product",
+            joinColumns = {@JoinColumn(name = "order_id", referencedColumnName = "order_id")},
+            inverseJoinColumns = {@JoinColumn(name = "product_id", referencedColumnName = "product_id")}
+    )
```

Comment:
change to
@JoinTable(
            name = "join_orders_product",
            joinColumns = {@JoinColumn(name = "product_id", referencedColumnName = "product_id")},
            inverseJoinColumns = {@JoinColumn(name = "order_id", referencedColumnName = "order_id")}
    )